docs/documenting/general-style-guide.md

```diff
@@ -26,6 +26,15 @@ A few of the most common cases picked up in reviews are listed below:
   - Less content makes tables easier to read.
 - When limiting the amount of text is not possible, consider using other data structures for representing the information. For example, lists or `{% configuration_basic %}` can be used.
 
+## Markdown
+
+A few notes about the use of markdown in this documentation:
+
+### Markdown lists
+
+- For unordered lists, use `-`, not `*`.
+- For ordered lists, use increasing numbers.
```

Comment:
since markdown automatically numbers the lists, is this required?

what I mean is that 
```
1. One
1. Two
1. Three
```

results in

1. One
1. Two
1. Three


That makes it easier when there is a big list of points, since you don't have to change numbers manually.

Reply:
I know, but I think the linter doesn't like it.